informative_sampling/Sample[] samples
informative_sampling/Bounds bounds
float64 noise_value
bool noise_learned
int64[] dims
int64 quantity_index
---
std_msgs/Float64MultiArray belief_map
std_msgs/Float64MultiArray uncertainty_map
